### ISSUE TITLE
Translating with no test set can result in drafts in the wrong language

### ISSUE THREAD

**Issue by mmartin9684-sil** [sillsdev/silnlp#578]
With the new learning rate schedule and much faster training runs, it's possible to do onboarding with 2 training runs - a first run with typical train/val/test splits to assess the model quality, and a second run with only a train split to bump the model quality by using all available data for training (similar to Serval).  However, on this second model, if the experiment also uses the "--translate" option, the drafts can sometimes include content in the wrong target language.  For example, some parts of a Javanese draft were translated into English.

It appears that the translate step may rely on the test set to understand the target language, and can't specify it properly if there is no test set.

A workaround is to explicitly set the trg_iso in the translate_config.yml:
`translate: 
    books: MAT
    src_project: NIV11R
    checkpoint: last
    trg_iso: jav_Latn`
